libs/api/domains/application/src/lib/dto/getAttachmentPresignedUrl.input.ts

```diff
@@ -0,0 +1,12 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString, IsEnum } from 'class-validator'
+@InputType()
+export class GetAttachmentPresignedUrlInput {
+  @Field(() => String)
+  @IsString()
+  id!: string
+
+  @Field(() => String)
+  @IsString()
```

Comment:
You can remove IsString ... doesn't do anything for the schema.
class-validator should be removed as well.